product_docs/docs/pgd/4/limitations.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: "Limitations"
+---
+
+
+This section covers design limitations of BDR, that should be taken into account
+when planning your deployment.
+
+## Limits
+
+- BDR can run hundreds of nodes on good-enough hardware and network. However, for mesh-based deployments, we generally don't recommend running more than 32 nodes in one cluster. Each master node can be protected by multiple physical or logical standby nodes. There's no specific limit on the number of standby nodes, but typical usage is to have 2&ndash;3 standbys per master. Standby nodes don't add connections to the mesh network, so they aren't included in the
```

Comment:
This sentence is heavily outdated, a) we don't recommend physical standbys at all, b) we generally don't recommend logical standbys other than when user really wants to save on cross-region bandwidth, and typical usage is to not use any standbys or any other special kind of node and just have data nodes.

Reply:
Suggesting:

We recommend that generally users only run data nodes. Standby node configurations are possible, but are only appropriate for situations where inter-regional bandwidth costs are a concern.  

or:

We recommend that generally users only run data nodes. Use of standby nodes is supported but not recommended.

Reply:
New language applied.